core/services/eth/log_broadcaster.go

```diff
@@ -84,43 +84,39 @@ func NewLogBroadcaster(ethClient Client, orm ormInterface, backfillDepth uint64)
 // for determining whether or not the log has been consumed and for marking
 // the log as consumed
 type LogBroadcast interface {
-	Log() Log
-	UpdateLog(Log)
+	DecodedLog() interface{}
```

Comment:
I didn't really change this from the previous implementation. Previously `Log` was just an interface that we were already doing a type switch on in the FM. I just separated the `DecodedLog` and the `RawLog` into two different getters, rather than having them nested. Hard to know what `DecodedLog` will be in the future since those types are generated from abigen.